explorer/go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/ignite/apps/explorer
 
-go 1.21.1
+go 1.22.2
```

Comment:
If possible it might be good to keep the version low, unless a dependency or new feature requires to bump it